tests/keras_tests/feature_networks_tests/feature_networks/mixed_precision_bops_test.py

```diff
@@ -38,6 +39,10 @@ def __init__(self, unit_test, mixed_precision_candidates_list):
 
         self.mixed_precision_candidates_list = mixed_precision_candidates_list
 
+    def get_core_config(self):
+        return CoreConfig(quantization_config=QuantizationConfig(
```

Comment:
I don't know if you want to change this, but "get" should be "generate" because you generate a core  config, you don't fetch it from a list or dict

Reply:
not as part of this PR (hopefully we'll get to a global tests refactor soon)